jenkins/Jenkinsfile.build

```diff
@@ -63,7 +63,7 @@ pipeline {
                     }
                 }
                 sh """
-                    aws s3 rm "${targetDomain}" --recursive --exclude "nhd_order_tiles/*" --exclude "tiles/*" --exclude "basetiles/*" --exclude "openmaptiles/*"
+                    aws s3 rm "${targetDomain}" --recursive --exclude "nhd_order_tiles/*" --exclude "tiles/*" --exclude "basetiles/*" --exclude "openmaptiles/*" --exclude "nhd_order_tiles/*"
```

Comment:
Alternatively, could try doing exclude everything here, then including just the relevant vue parts?  Maybe that would speed things up?

Reply:
I beat you to it. The exclude is the first one in the line.  Interesting idea on the 'include' statement., although the point will be moot once the 'tile server' is up. Then we will no longer need the base layers in our application buckets.

Reply:
Lol, guess I missed that...